demo/index.html

```diff
@@ -63,7 +63,7 @@ <h2 id="iene-heading">initial element, no escape</h2>
         Here is a focus trap <a href="#">with</a> <a href="#">some</a> <a href="#">focusable</a> parts.
       </p>
       <p>
-        <label htmlFor="focused-input" class="inline-label">
+        <label for="focused-input" class="inline-label">
```

Comment:
`htmlFor` is not a valid attribute for `<label>` but instead `for` should be used so the cypress accessibility 
 selector `findByRole` works

Reply:
For sure. `htmlFor` is the React equivalent because `for` is seen as a JS statement in JSX. Force of habit for whoever put that in there. 😉  And good catch by you!